--- xrdp-0.9.16/xrdp/xrdp.ini.orig
+++ xrdp-0.9.16/xrdp/xrdp.ini
@@ -190,22 +190,22 @@
 ; Some session types such as Xorg, X11rdp and Xvnc start a display server.
 ; Startup command-line parameters for the display server are configured
 ; in sesman.ini. See and configure also sesman.ini.
-[Xorg]
-name=Xorg
-lib=libxup.so
-username=ask
-password=ask
-ip=127.0.0.1
-port=-1
-code=20
+#[Xorg]
+#name=Xorg
+#lib=libxup.so
+#username=ask
+#password=ask
+#ip=127.0.0.1
+#port=-1
+#code=20

-[Xvnc]
-name=Xvnc
-lib=libvnc.so
-username=ask
-password=ask
-ip=127.0.0.1
-port=-1
+#[Xvnc]
+#name=Xvnc
+#lib=libvnc.so
+#username=ask
+#password=ask
+#ip=127.0.0.1
+#port=-1
 #xserverbpp=24
 #delay_ms=2000
 ; Disable requested encodings to support buggy VNC servers
@@ -216,6 +216,21 @@
 ; display number of the session
 #chansrvport=DISPLAY(0)

+[neutrinordp-any]
+name=neutrinordp-any
+lib=libxrdpneutrinordp.so
+ip=ask
+port=ask3389
+username=ask
+password=ask
+#pamusername=asksame
+#pampassword=asksame
+#pamsessionmng=127.0.0.1
+#delay_ms=2000
+; Currently NeutrinoRDP doesn't support dynamic resizing. Uncomment
+; this line if you're using a client which does.
+#enable_dynamic_resizing=false
+
 [vnc-any]
 name=vnc-any
 lib=libvnc.so
@@ -227,17 +242,6 @@
 #pampassword=asksame
 #pamsessionmng=127.0.0.1
 #delay_ms=2000
-
-[neutrinordp-any]
-name=neutrinordp-any
-lib=libxrdpneutrinordp.so
-ip=ask
-port=ask3389
-username=ask
-password=ask
-; Currently NeutrinoRDP doesn't support dynamic resizing. Uncomment
-; this line if you're using a client which does.
-#enable_dynamic_resizing=false

 ; You can override the common channel settings for each session type
 #channel.rdpdr=true

