--- a/xrdp/xrdp.ini.in
+++ b/xrdp/xrdp.ini.in
@@ -193,48 +193,70 @@
 ; Some session types such as Xorg, X11rdp and Xvnc start a display server.
 ; Startup command-line parameters for the display server are configured
 ; in sesman.ini. See and configure also sesman.ini.
-[Xorg]
-name=Xorg
-lib=libxup.so
-username=ask
-password=ask
-ip=127.0.0.1
-port=-1
-code=20
+#[Xorg]
+#name=Xorg
+#lib=libxup.so
+#username=ask
+#password=ask
+#ip=127.0.0.1
+#port=-1
+#code=20
 
-[Xvnc]
-name=Xvnc
-lib=libvnc.so
-username=ask
-password=ask
-ip=127.0.0.1
-port=-1
+#[Xvnc]
+#name=Xvnc
+#lib=libvnc.so
+#username=ask
+#password=ask
+#ip=127.0.0.1
+#port=-1
 #xserverbpp=24
 #delay_ms=2000
 ; Disable requested encodings to support buggy VNC servers
 ; (1 = ExtendedDesktopSize)
 #disabled_encodings_mask=0
 
+[RDP-Porxy]
+name=RDP-Proxy
+lib=libxrdpneutrinordp.so
+ip=ask
+port=ask3389
+username=ask
+password=ask
 
-[vnc-any]
-name=vnc-any
+# [RDP-Proxy-PAM]
+# name=RDP-Proxy with PAM auth
+# lib=libxrdpneutrinordp.so
+# ip=ask192.168.0.101
+# port=3389
+# username=askWin-user
+# password=askWin-password
+# pamusername=askLinuxPAM-User
+# pampassword=askLinuxPAM-password
+# pamsessionmng=127.0.0.1
+# delay_ms=2000
+
+; Note: PAM user should be members of the "tsusers" group.
+;       See also sesman.ini
+
+[VNC-Proxy]
+name=VNC-Proxy
 lib=libvnc.so
 ip=ask
 port=ask5900
 username=na
 password=ask
-#pamusername=asksame
-#pampassword=asksame
-#pamsessionmng=127.0.0.1
-#delay_ms=2000
 
-[neutrinordp-any]
-name=neutrinordp-any
-lib=libxrdpneutrinordp.so
-ip=ask
-port=ask3389
-username=ask
-password=ask
+# [VNC-Proxy-PAM]
+# name=VNC-Proxy with PAM auth
+# lib=libvnc.so
+# ip=ask192.168.0.101
+# port=ask5900
+# username=na
+# password=askVNC-password
+# pamusername=askLinuxPAM-User
+# pampassword=askLinuxPAM-password
+# pamsessionmng=127.0.0.1
+# delay_ms=2000
 
 ; You can override the common channel settings for each session type
 #channel.rdpdr=true
